tests/publiccloud/migration.pm

```diff
@@ -21,6 +21,7 @@ use utils;
 use publiccloud::utils;
 
 our $target_version = get_required_var('TARGET_VERSION');
+our $not_clean_vm = get_required_var('PUBLIC_CLOUD_NO_CLEANUP_ON_FAILURE');
```

Comment:
```suggestion
our $not_clean_vm = get_var('PUBLIC_CLOUD_NO_CLEANUP_ON_FAILURE');
```
`get_required_var` will fail test if variable is not defined , I don't think this is necessary

Reply:
Done

Reply:
done